src/frisby/expects.js

```diff
@@ -11,7 +11,7 @@ const utils = require('./utils');
  * Jasmine and others
  */
 function incrementAssertionCount() {
-  if (_.isFunction(expect)) {
+  if (typeof expect != 'undefined' && _.isFunction(expect)) {
```

Comment:
@OrangeBean 

If expect is undefined then _.isFunction(expect) returns false.
https://lodash.com/docs/#isFunction
So, I think there is no effect.

In this issue, what is set for expect ?

Reply:
`expect` is not defined anywhere as there is no Jasmine. The issue is that it is not possible to call function with undefined variable as a parameter.

Reply:
@OrangeBean 
Thanks. I understand.

use !== is more better.